main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+)
+
+type config struct {
+	InputFilePath  string          `env:"encrypted_file_path,file"`
+	OutputFilePath string          `env:"output_file_path,required"`
+	Passphrase     stepconf.Secret `env:"decrypt_passphrase,required"`
+}
+
+func failf(format string, args ...interface{}) {
+	log.Errorf(format, args...)
+	os.Exit(1)
+}
+
+func main() {
+	var cfg config
+	if err := stepconf.Parse(&cfg); err != nil {
+		failf("Could not create config: %v\n", err)
+	}
+	stepconf.Print(cfg)
+	outputFileAbsPath, err := pathutil.AbsPath(cfg.OutputFilePath)
+	if err != nil {
+		failf("Converting to absolute path failed, error: %#v", err)
+	}
+	if err := os.Remove(outputFileAbsPath); err != nil && !os.IsNotExist(err) {
+		failf("Output file already exists, removal failed: error: %#v", err)
+	}
+
+	log.Infof("Decrypting file...")
+	cmdSlice := []string{"gpg", "--batch", "--passphrase", cfg.Passphrase.String(),
+		"--output", outputFileAbsPath, "--decrypt", cfg.InputFilePath}
+	log.Donef("=> %s", command.PrintableCommandArgs(true, cmdSlice))
+	// Replace obfuscated passphrase with real one
+	cmdSlice[3] = string(cfg.Passphrase)
+	model, err := command.NewFromSlice(cmdSlice)
+	if err != nil {
+		failf("Creating command failed, error: %#v", err)
+	}
+	out, err := model.RunAndReturnTrimmedCombinedOutput()
+	if err != nil {
```

Comment:
Please have a look at this solution: https://github.com/bitrise-steplib/steps-xcode-build-for-test/blob/master/main.go#L270-L277

If the error is not nil:
- may performing the given command has an issue, in this case, the error describes the problem
- may the command failed itself, this case the error will hold an exit status error, like: `exit status 1`, in this case, the error is not too descriptive, but the command's output should be descriptive